examples/09-DeepTCN-examples.ipynb

```diff
@@ -44,7 +44,7 @@
     "from darts import TimeSeries\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        series=concatenate([train_en_transformed, val_en_transformed]),
</pre><p>Let's define <code>series_en_transformed</code> again before (where we apply the scaler), and then reuse it here and in the next cell. Like this we can avoid having to compute series_en_transformed twice, which can reduce time for longer series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>